### DISCUSSION TITLE
why collabrators is not showing in my repo

### DISCUSSION THREAD

**Discussion by shivamsi0006** [community/community#103120] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

asking

**Comment by prashantsah** [comment]
Just invite them from setting in repo

**Comment by ghostinhershell** [comment]
Hi @shivamsi0006 and @prashantsah,

Two things to note here:

GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community.

Furthermore, the GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same content in multiple places. As a result, we have deleted the duplicate post, and we ask that you no longer cross-post.

Again, for additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

Thanks for understanding.
